README.adoc

```diff
@@ -121,9 +121,7 @@ set the `io_openliberty_guides_system_inMaintenance` configuration property
 to `false` in the `resources/CustomConfigSource.json` file.
 Otherwise, the service returns a `Status.SERVICE_UNAVAILABLE` message, which makes it unavailable.
 
-Next, the `src/main/java/io/openliberty/guides/inventory/client/SystemClient.java` class
-includes the `getPropertiesHelper()` method to make a request to the
-`system` service and check for the response.
+Next, the `src/main/java/io/openliberty/guides/inventory/client/ExceptionMapper.java` class includes the `getPropertiesHelper()` method to make a request to the	+includes the `handles()` and  `toThrowable()` methods to check for the response in order to handle the exceptions.
```

Comment:
This sentence isn't correct, ie, "to the	+includes the.."

Reply:
@justineechen can you handle this one?

Reply:
@justineechen  We don't necessarily need to explain the `ExceptionMapper.java` in this guide, it confuses readers somehow.